### ISSUE TITLE
Design Recipe rendering glitches

### ISSUE THREAD

**Issue by schanzer** [bootstrapworld/curriculum#163]
- [x] In Pyret, add `#` to contract
- [x] In Pyret DRs, keywords like `fun`, `is`, and `end` should be bold
- [x] See if there's a better way to handle multiline examples
- [x] Try to conserve vertical space, especially when examples bleed onto two lines

**Comment by schanzer** [comment]
I have a solution for multiline examples:
```<div class="openblock wrapper recipe_example_line">
<div class="content">
<div class="paragraph">
<p><span class="clear"> </span>
<span class="clear"> </span><span class="studentAnswerCode blank">MM</span><span class="studentAnswer studentAnswerCode solution recipe_name">nametag</span> <span class="bigparen">(</span><span class="studentAnswer studentAnswerCode solution recipe_example_inputs">animalA</span><span class="bigparen">)</span>
<span class="is_recipe_example_body"><span class="studentAnswerCode blank">MM</span><span class="pyretkeyword">is</span> <span class="studentAnswer studentAnswerCode solution recipe_example_body">text(animalA["name"], 10, "purple")</span></span>
<span class="clear"> </span></p>
</div>
</div>
</div>```

**Comment by schanzer** [comment]
Implemented the new DOM in an experiment push in 6ab4b58c71409342db8ca84bdf3c4bc377cbcb64. Will close if Dorai accepts it.
